Sources/LiveViewNative/LVN.swift

```diff
@@ -0,0 +1,16 @@
+//
+//  LVN.swift
+//  LiveViewNative
+//
+//  Created by Dylan.Ginsburg on 4/28/23.
+//
+
+import Foundation
+
+struct LVN {
+    static func JSONDecoder() -> JSONDecoder {
+        let decoder = Foundation.JSONDecoder()
+        decoder.keyDecodingStrategy = .convertFromSnakeCase
+        return decoder
+    }
+}
```

Comment:
I didn't see an existing place for something like this. I like the syntax because it namespaces the construction: `LVN.JSONDecoder()`. But it can be done in whatever way fits best in this project.

Reply:
This might be better as just a global variable, like:
```swift
let snakeCaseDecoder: JSONDecoder = {
  ...
}()
```
The module provides the namespace, like `LiveViewNative.snakeCaseDecoder` and the instance can be shared instead of creating and configuring it every time its used.

Reply:
@carson-katri 

The reason I didn't use a single instance of JSONDecoder is that I don't see a guarantee that it's thread safe to possibly have parallel decodings being done by that same instance. The Apple docs say it's "reusable", which isn't necessarily the same as re-entrant. I looked it up and SO discussions do say it's thread safe as implemented today. But that's a Swift library implementation detail and if it's not promised by the docs I chose to be safe

The performance difference is going to be negligible so my suggestion would be to do it the safe way and optimize later when the project is more complete with more tests.

As for the name, I thought that having `JSONDecoder` remain in the name would make it easy for people to understand what it is. You see that and you know what it is; instead of having to go look it up. But I can see why you don't like having a fake namespace when the library already has a real one.

How about a global function that retains the name and requires the real namespace to invoke?

```swift
func JSONDecoder() -> JSONDecoder {
    let decoder = Foundation.JSONDecoder()
    decoder.keyDecodingStrategy = .convertFromSnakeCase
    return decoder
}

LiveViewNative.JSONDecoder().decode(...)
```

In the future, the implementation of `LiveViewNative.JSONDecoder()` could be changed to use a single instance if that proves worthwhile.

Or if you feel strongly about the global var and single instance I'll change it to what you wrote above. You know the full picture better than I do.

Reply:
That works. 👍🏻 I changed it.

Reply:
Thread safety isn't important here, modifiers are always decoded on the main thread. Although we're not currently, I think it is worth reusing the JSONDecoder since it's trivially possible.

Reply:
It's now using a single instance wrapped by `makeJSONDecoder()`.